### PR TITLE
Fancy ouput for targz()

### PR DESCRIPTION
**Pull request by heitorPB**
added some text about the size of the tar and the tar.gz files:

> $ targz aclog
aclog/
[...]
aclog/aclog-20160208-2343(0).log
Compressing .tar (3020 kb) using `gzip`…
aclog.tar:	 78.4% -- replaced with aclog.tar.gz
aclog.tar.gz (651 kb) created successfully.

### CODE REVIEW

File: .functions [hunk 1]
```diff
@@ -48,10 +48,16 @@ function targz() {
 		fi;
 	fi;
 
-	echo "Compressing .tar using \`${cmd}\`…";
+	echo "Compressing .tar ($((size / 1000)) kb) using \`${cmd}\`…";
```
Comment (mathiasbynens) [inline .functions]: Should be `kB` instead of `kb`

File: .functions [hunk 2]
```diff
@@ -48,10 +48,16 @@ function targz() {
...
 	"${cmd}" -v "${tmpFile}" || return 1;
 	[ -f "${tmpFile}" ] && rm "${tmpFile}";
-	echo "${tmpFile}.gz created successfully.";
+
+	zippedSize=$(
+		stat -f"%z" "${tmpFile}.gz" 2> /dev/null; # OS X `stat`
+		stat -c"%s" "${tmpFile}.gz" 2> /dev/null # GNU `stat`
```
Comment (mathiasbynens) [inline .functions]: missing `;`

File: .functions [hunk 3]
```diff
@@ -48,10 +48,16 @@ function targz() {
...
+	);
+
+	echo "${tmpFile}.gz ($((zippedSize / 1000)) kb) created successfully.";
```
Comment (mathiasbynens) [inline .functions]: Same here.

**Comment by mathiasbynens** [comment]
Could you rebase please?

**Comment by heitorPB** [comment]
Sorry, but where are the missing `;` ?

**Comment by mathiasbynens** [comment]
https://github.com/mathiasbynens/dotfiles/pull/670/files#r65564115

**Comment by heitorPB** [comment]
So there was also `;` missing in the original code?

**Comment by mathiasbynens** [comment]
Thanks!

**Pull request merged**